prometheus/prom_rules/prometheus.rules.yml

```diff
@@ -261,3 +261,24 @@ groups:
     annotations:
       description: 'OOM Kill on {{ $labels.instance }}'
       summary: A process was terminated on Instance {{ $labels.instance }}
+  - alert: tooManyFiles
+    expr: (node_filesystem_files - node_filesystem_files_free) / on(instance) group_left count(scylla_reactor_cpu_busy_ms) by (instance)>20000
```

Comment:
This and the following expressions are going to check every mountpoint on every node. This can potentially generate false alarms. 

For example, I see that we filter by a hard coded `mountpoint="/var/lib/scylla"` in a DiskFull family of alerts. Why don't we do the same here?

Reply:
I had this comment on the issue itself, I think that if we have more than 20K per shard on any mountpoint, it's still a valid alert, but I'll be happy to change that

Reply:
Do we care about other mountpoints? I thought that the issue is related to the OOM in scylla that may be caused if it needs to open that many files.

In general some users (and there are a few that do that) have different mount points for `data` and `commitlog` for example.

However the OOM issue I've referenced is mainly about `sstables` hence the mount point of the main interest is `data`. 

Ideally it would be "auto-detectable". But if it's impossible/too hard - a hard coded value can do as well as long as we document it and the user knows that it needs to be adjusted if he/she decides to use non-default mount options.

On the other hand checking everything (as in this PR right now) can be used as a default too as long as we document and provide an example how to filter if needed.

Reply:
@vladzcloudius I don't think there's a good way of finding that mount point if it's not reported by Scylla. I think I'll need to add a larger section around alerts which will cover existing alerts, how to customize and add alerts.